packages/atomic/src/components/search/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -25,6 +25,8 @@ import {Bindings} from '../../atomic-search-interface/atomic-search-interface';
 
 /**
  * @internal
+ * An `atomic-segmented-facet` displays a facet of the results for the current query in a horizontal aspect
```

Comment:
```suggestion
 * The `atomic-segmented-facet` displays a facet of the results for the current query in a horizontal aspect.
```
(not sure what you mean by "horizontal aspect" though 🤔)

Reply:
The segmented facets differ from the regular facets by the way they're displayed. The segmented facets are shown in a horizontal aspect/layout

<img width="1013" alt="Screen Shot 2022-06-28 at 11 13 00 AM" src="https://user-images.githubusercontent.com/97299176/176215645-228eff8a-297b-440a-a304-178027103407.png">

vs

The regular facets' vertical and collapsable aspect

<img width="356" alt="Screen Shot 2022-06-28 at 11 13 53 AM" src="https://user-images.githubusercontent.com/97299176/176215809-d41ebddb-013a-4e68-a43f-1f2e7fa65c9e.png">